### DISCUSSION TITLE
which event do we use when we want to track everytime we draw the polyline on gmap vue

### DISCUSSION THREAD

**Discussion by fathurqarnain** [diegoazh/gmap-vue#305] [Q&A]
[

https://github.com/diegoazh/gmap-vue/assets/103421451/544bcce8-d9ef-42cc-bf86-bfc437303499

](url)
hi guys so i want to track everytime we click / draw the polyline on gmap drawing manager on vue, i already use all the event available but cannot get the draw event, here is my code

![Screenshot from 2023-10-04 02-31-20](https://github.com/diegoazh/gmap-vue/assets/103421451/3864ca97-5848-4cd3-a741-3a28b7b55cc2)

is there any way to do that?
thanks before

**Comment by diegoazh** [comment]
Hi, sorry for being late, I just saw this today. You can use the [`dragend`](https://developers.google.com/maps/documentation/javascript/reference/polygon?hl=es#Polyline.dragend) event.
